app/timetables/models.py

```diff
@@ -39,3 +39,20 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Dish(ForceCapitalizeMixin, TimestampMixin):
+    """
+    Model representing Dish.
+    A dish represents the actual food served as a given
+    course under an option of a meal on a cycle day in a timetable.
+    E.g, Coconut rice garnished with fish stew and chicken or just
+    Ice-cream.
+    """
+
+    name = models.CharField(max_length=255, unique=True)
+
```

Comment:
@andela-joyebanji I know this wasn't initially on the card, but someone mentioned that the Dish model should have a `description` field where we can put any extra information about the dish that may be required e.g dietary and nutritional info, calorie count, etc.
cc @andela-kanyanwu